definitions/install.yaml

```diff
@@ -3848,8 +3848,6 @@ spec:
     spec:
       containers:
       - args:
-        - --filterK8sResources=[Event,*,*][*,kube-system,*][*,kube-public,*][*,kube-node-lease,*][Node,*,*][APIService,*,*][TokenReview,*,*][SubjectAccessReview,*,*][*,kyverno,*][Binding,*,*][ReplicaSet,*,*][ReportChangeRequest,*,*][ClusterReportChangeRequest,*,*][PolicyReport,*,*][ClusterPolicyReport,*,*]
-        - -v=2
```

Comment:
Do not remove the log level here.

Reply:
as was mention by @realshuting 
https://github.com/kyverno/kyverno/issues/2347#issuecomment-910970466
I thought we can remove it

Reply:
If user want to change the log level while running kyverno this log level is necessary.

@realshuting - your thoughts on this...

Reply:
It doesn't change anything to remove it, but it's good to have to allow users to simply increment the integer to enable more verbose logs without having to add the argument entirely. I suggest leaving it.

Reply:
@slayer321 - do not remove the log level from here.

Reply:
ok, will not remove it.

Reply:
Let me know when you are done with the changes.